pyscript.core/tests/integration/test_02_display.py

```diff
@@ -72,6 +73,79 @@ def test_target_parameter(self):
         mydiv = self.page.locator("#mydiv")
         assert mydiv.inner_text() == "hello world"
 
+    def test_target_parameter_with_sharp(self):
+        self.pyscript_run(
+            """
+            <script type="py">
+                from pyscript import display
+                display('hello world', target="#mydiv")
+            </script>
+            <div id="mydiv"></div>
+            """
+        )
+        mydiv = self.page.locator("#mydiv")
+        assert mydiv.inner_text() == "hello world"
+
+    def test_non_existing_id_target_raises_value_error(self):
+        self.pyscript_run(
+            """
+            <script type="py">
+                from pyscript import display
+                display('hello world', target="non-existing")
+            </script>
+            """
+        )
+        error_msg = (
+            f"Invalid selector with id=non-existing. Cannot be found in the page."
+        )
+        self.check_py_errors(f"ValueError: {error_msg}")
+        # Check Console error
+        tb_lines = self.console.error.lines[-1].splitlines()
+        assert tb_lines[0] == "PythonError: Traceback (most recent call last):"
```

Comment:
I think this is not strictly necessary: if we decide that Python exceptions should go *both* to the console and to the DOM (and I think they should), then this check should go inside `check_py_errors`, probably.